### PR TITLE
Finished Bubble Sort

### PR DESCRIPTION
**Pull request by IvanDerlich**

### CODE REVIEW

File: buble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                aux = array[i+1]
+                array[i+1] = val
+                array[i]= aux
+            end
+        end
+    end     
+end
+  
+bubble_sort([4,3,78,2,0,2])
+bubble_sort_by([4,3,78,2,0,2]) do |left, right|
```
Comment (JamesOkunlade) [inline buble_sort.rb]: you are to test with ["HI", "Hello", "Hey"] as input.

File: buble_sort.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                array[i+1] = val
+                array[i]= aux
+            end
+        end
+    end     
+end
+  
+bubble_sort([4,3,78,2,0,2])
+bubble_sort_by([4,3,78,2,0,2]) do |left, right|        
+    right < left
```
Comment (JamesOkunlade) [inline buble_sort.rb]: it's string you're passing into the block so this should be right.length and left.length

**Comment by IvanDerlich** [comment]
I compare the index to avoid running the comparison on the last element. 
Is there another way to deal with this issue?

**Comment by IvanDerlich** [comment]
Thanks James! It's always a pleasure working with you! 
See you in another oportunity.
Or read you?

**Pull request merged**